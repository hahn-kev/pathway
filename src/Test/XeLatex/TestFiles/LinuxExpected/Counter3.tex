\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\sectionletter="Charis SIL" at 12pt
\font\articlesectionletter="Charis SIL/I" at 12pt
\font\sensearticlesectionletter="Charis SIL/BI":color=0000ff at 12pt
\font\sensebeforearticlesectionletter="Charis SIL/BI":color=ff0000 at 12pt
\font\firstoftypegrammarcategorylastoftypesensearticlesectionletter="Charis SIL/BI":color=0000ff at 12pt
\font\definitionfirstoftypelastoftypesensearticlesectionletter="Charis SIL/BI":color=0000ff at 12pt
\font\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter="Charis SIL/BI":color=0000ff at 12pt
\font\examplesensearticlesectionletter="Charis SIL/BI":color=0000ff at 12pt
\font\firstoftypelastoftypesubentryarticlesectionletter="Charis SIL/I" at 12pt
\font\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter="Charis SIL/I" at 12pt
\font\examplefirstoftypelastoftypesubentryarticlesectionletter="Charis SIL/I" at 12pt
\font\firstoftypeheadwordlastoftypearticlesectionletter="Charis SIL/I" at 12pt
\font\firstoftypelastoftypepronunciationarticlesectionletter="Charis SIL/I" at 12pt
\font\firstoftypelastoftypestressfirstoftypelastoftypepronunciationarticlesectionletter="Charis SIL/I" at 12pt
\font\firstoftypevariantinflectionarticlesectionletter="Charis SIL/I" at 12pt
\font\variantinflectionarticlesectionletter="Charis SIL/I" at 12pt
\font\lastoftypevariantinflectionarticlesectionletter="Charis SIL/I" at 12pt
\font\exampleusefirstoftypelastoftypesensearticlesectionletter="Charis SIL/BI":color=0000ff at 12pt

\newpage 
\thispagestyle{empty} 
\mbox{} 
\newpage 
\newpage 
\setcounter{page}{1} 
\pagenumbering{arabic} 
\pagestyle{fancy} 

\sensebeforearticlesectionletter{1.1}\sensearticlesectionletter{1 }\firstoftypegrammarcategorylastoftypesensearticlesectionletter{N-Count}\definitionfirstoftypelastoftypesensearticlesectionletter{A }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter{waitress}\definitionfirstoftypelastoftypesensearticlesectionletter{ is a woman who works in a restaurant, serving people with food and drink.}\sensebeforearticlesectionletter{1.2}\sensearticlesectionletter{2 }\firstoftypegrammarcategorylastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypesensearticlesectionletter{A woman who }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter{waitresses}\definitionfirstoftypelastoftypesensearticlesectionletter{ works in a restaurant serving food and drink.}\examplesensearticlesectionletter{She had been working in a pub, cooking and waitressing.}\firstoftypegrammarcategorylastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypesensearticlesectionletter{A woman who }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter{waitresses}\definitionfirstoftypelastoftypesensearticlesectionletter{ works in a restaurant serving food and drink.}\examplesensearticlesectionletter{She had been working in a pub, cooking and waitressing.}\firstoftypegrammarcategorylastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypesensearticlesectionletter{A woman who }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter{waitresses}\definitionfirstoftypelastoftypesensearticlesectionletter{ works in a restaurant serving food and drink.}\examplesensearticlesectionletter{She had been working in a pub, cooking and waitressing.}\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter{wait}\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter{ress}\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter{ing}\examplefirstoftypelastoftypesubentryarticlesectionletter{She does a bit of waitressing as a part-time job.}\section*{\needspace {8\baselineskip}{\firstoftypeheadwordlastoftypearticlesectionletter{waive}}}\section*{\needspace {8\baselineskip}{\firstoftypelastoftypepronunciationarticlesectionletter{w}}}\section*{\needspace {8\baselineskip}{\firstoftypelastoftypestressfirstoftypelastoftypepronunciationarticlesectionletter{eÉª}}}\section*{\needspace {8\baselineskip}{\firstoftypelastoftypepronunciationarticlesectionletter{v}}}\section*{\needspace {8\baselineskip}{\firstoftypevariantinflectionarticlesectionletter{waives}}}\section*{\needspace {8\baselineskip}{\variantinflectionarticlesectionletter{waiving}}}\section*{\needspace {8\baselineskip}{\lastoftypevariantinflectionarticlesectionletter{waived}}}\sensebeforearticlesectionletter{2.1}\sensearticlesectionletter{1 }\firstoftypegrammarcategorylastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypesensearticlesectionletter{If you }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter{waive}\definitionfirstoftypelastoftypesensearticlesectionletter{ your right to something, for example legal representation, you choose not to have it or do it.}\exampleusefirstoftypelastoftypesensearticlesectionletter{V n}\examplesensearticlesectionletter{He pleaded guilty to the murders of three boys and waived his right to appeal.}\sensebeforearticlesectionletter{2.2}\sensearticlesectionletter{2 }\firstoftypegrammarcategorylastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypesensearticlesectionletter{If someone }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypesensearticlesectionletter{waives}\definitionfirstoftypelastoftypesensearticlesectionletter{ a rule, they say that people do not have to obey it in a particular situation.}\exampleusefirstoftypelastoftypesensearticlesectionletter{V n}\examplesensearticlesectionletter{The art gallery waives admission charges on Sundays.}

\end{document}
