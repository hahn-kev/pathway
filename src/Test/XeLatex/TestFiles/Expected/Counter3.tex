\documentclass[a4paper]{article} 
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage{alphalph}
\usepackage[left=2cm,right=2cm,top=1.99cm,bottom=1.99cm,includeheadfoot]{geometry}
\usepackage{changepage}
\usepackage{needspace}

\parindent=0pt
\parskip=\medskipamount
\begin{document}
\pagestyle{plain}
\sloppy
\setlength{\parfillskip}{0pt plus 1fil}
\font\sectionletter="Times New Roman/GR" at 12pt
\font\articlesectionletter="Times New Roman/GR/I" at 12pt
\font\firstoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\firstoftypegrammarcategorylastoftypefirstoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\definitionfirstoftypelastoftypefirstoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypefirstoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\lastoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\firstoftypegrammarcategorylastoftypelastoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypelastoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\examplefirstoftypelastoftypelastoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\firstoftypelastoftypesubentryarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\examplefirstoftypelastoftypefirstoftypelastoftypesubentryarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\firstoftypeheadwordlastoftypearticlesectionletter="Times New Roman/GR/I" at 12pt
\font\firstoftypelastoftypepronunciationarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\firstoftypelastoftypestressfirstoftypelastoftypepronunciationarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\firstoftypevariantinflectionarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\variantinflectionarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\lastoftypevariantinflectionarticlesectionletter="Times New Roman/GR/I" at 12pt
\font\exampleusefirstoftypelastoftypefirstoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\examplefirstoftypelastoftypefirstoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt
\font\exampleusefirstoftypelastoftypelastoftypesensearticlesectionletter="Times New Roman/GR/BI":color=ff0000 at 12pt

\mbox{} 
\newpage 
\newpage 
\setcounter{page}{1} 
\pagenumbering{arabic} 
\pagestyle{fancy} 

\firstoftypesensearticlesectionletter{1.1}\firstoftypesensearticlesectionletter{1 }\firstoftypegrammarcategorylastoftypefirstoftypesensearticlesectionletter{N-Count}\definitionfirstoftypelastoftypefirstoftypesensearticlesectionletter{A }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypefirstoftypesensearticlesectionletter{waitress}\definitionfirstoftypelastoftypefirstoftypesensearticlesectionletter{ is a woman who works in a restaurant, serving people with food and drink.}\lastoftypesensearticlesectionletter{1.2}\lastoftypesensearticlesectionletter{2 }\firstoftypegrammarcategorylastoftypelastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{A woman who }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypelastoftypesensearticlesectionletter{waitresses}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{ works in a restaurant serving food and drink.}\examplefirstoftypelastoftypelastoftypesensearticlesectionletter{She had been working in a pub, cooking and waitressing.}\firstoftypegrammarcategorylastoftypelastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{A woman who }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypelastoftypesensearticlesectionletter{waitresses}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{ works in a restaurant serving food and drink.}\examplefirstoftypelastoftypelastoftypesensearticlesectionletter{She had been working in a pub, cooking and waitressing.}\firstoftypegrammarcategorylastoftypelastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{A woman who }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypelastoftypesensearticlesectionletter{waitresses}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{ works in a restaurant serving food and drink.}\examplefirstoftypelastoftypelastoftypesensearticlesectionletter{She had been working in a pub, cooking and waitressing.}\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter{wait}\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter{ress}\firstoftypeheadwordlastoftypefirstoftypelastoftypesubentryarticlesectionletter{ing}\examplefirstoftypelastoftypefirstoftypelastoftypesubentryarticlesectionletter{She does a bit of waitressing as a part-time job.}\firstoftypeheadwordlastoftypearticlesectionletter{waive}\firstoftypelastoftypepronunciationarticlesectionletter{w}\firstoftypelastoftypestressfirstoftypelastoftypepronunciationarticlesectionletter{eÉª}\firstoftypelastoftypepronunciationarticlesectionletter{v}\firstoftypevariantinflectionarticlesectionletter{waives}\section*{\needspace {8\baselineskip}{\variantinflectionarticlesectionletter{waiving}}}\section*{\needspace {8\baselineskip}{\lastoftypevariantinflectionarticlesectionletter{waived}}}\firstoftypesensearticlesectionletter{2.1}\firstoftypesensearticlesectionletter{1 }\firstoftypegrammarcategorylastoftypefirstoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypefirstoftypesensearticlesectionletter{If you }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypefirstoftypesensearticlesectionletter{waive}\definitionfirstoftypelastoftypefirstoftypesensearticlesectionletter{ your right to something, for example legal representation, you choose not to have it or do it.}\exampleusefirstoftypelastoftypefirstoftypesensearticlesectionletter{V n}\examplefirstoftypelastoftypefirstoftypesensearticlesectionletter{He pleaded guilty to the murders of three boys and waived his right to appeal.}\lastoftypesensearticlesectionletter{2.2}\lastoftypesensearticlesectionletter{2 }\firstoftypegrammarcategorylastoftypelastoftypesensearticlesectionletter{Verb}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{If someone }\firstoftypelastoftypewordusedefinitionfirstoftypelastoftypelastoftypesensearticlesectionletter{waives}\definitionfirstoftypelastoftypelastoftypesensearticlesectionletter{ a rule, they say that people do not have to obey it in a particular situation.}\exampleusefirstoftypelastoftypelastoftypesensearticlesectionletter{V n}\examplefirstoftypelastoftypelastoftypesensearticlesectionletter{The art gallery waives admission charges on Sundays.}

\end{document}
